.github/workflows/nightly-test-integrations.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           # this is further going to multiplied in envoy-integration tests by the 
           # other dimensions in the matrix.  Currently TOTAL_RUNNERS would be
           # multiplied by 8 based on these values:
-          # envoy-version: ["1.25.11", "1.26.7", "1.27.3", "1.28.1"]
+          # envoy-version: ["1.26.8", "1.27.4", "1.28.2", "1.29.3"]
```

Comment:
We might want to hold off on the Envoy 1.29 upgrade here (and below changes) just bc there's a few other changes involved (docs, `consul-dataplane`).

For the last Envoy CVE, we ended up bumping for available fixes and just kept regression tests in place for EOL versions, which some operators may still be using: https://github.com/hashicorp/consul/pull/20589

Reply:
Thanks for the clarification and the review @zalimeni 🙏 I removed the Envoy 1.29 upgrade from this PR so we can plan and work on it in another PR. https://github.com/hashicorp/consul/pull/20956/commits/dbcfceeae847ecf44849d80a4f1a26e88ef034ad#diff-28333389a7c4d67672238a29243985aca62468300c0b6ad47302b49342771715